### PR TITLE
DOCS-456 Add Delete Account instructions

### PR DESCRIPTION
**Pull request by angelasimms**
Closes https://hazelcast.atlassian.net/browse/DOCS-456

### CODE REVIEW

File: docs/modules/ROOT/pages/delete-account.adoc [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+
+
+
+
+
+
+
+
+
```
Comment (EugeneAbramchuk) [inline docs/modules/ROOT/pages/delete-account.adoc:45]: Do we need all the extra new lines at the end?

File: docs/modules/ROOT/pages/delete-account.adoc [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+{description}
+
+
+[[cluster-and-data-deletion]]
+== Cluster and Data Deletion
+
+Deleting an account _removes_ all of the following:
+
+- Clusters linked to the account, including any associated data, such as data structures, custom classes, backups, and configuration.
+- Account and payment details, excluding your email address.
```
Comment (EugeneAbramchuk) [inline docs/modules/ROOT/pages/delete-account.adoc:17]: Do we need to provide a reason why do we keep the email?

File: docs/modules/ROOT/pages/delete-account.adoc (hunk 1 above)
Comment (angelasimms) [reply docs/modules/ROOT/pages/delete-account.adoc]: Lines removed.

File: docs/modules/ROOT/pages/delete-account.adoc (hunk 2 above)
Comment (angelasimms) [reply docs/modules/ROOT/pages/delete-account.adoc]: I've reworded and added a note. I've tried to separate out the retention of the users email address (in line with our privacy policy) and the deletion of the account.

**Review by EugeneAbramchuk — APPROVED** [review]
LGTM

**Pull request merged**